packages/@o3r/eslint-config/index.ts

```diff
@@ -0,0 +1,2 @@
+import typescriptConfig from './typescript';
```

Comment:
should be in a `src/` folder with the other `.ts` and `.js` from this root folder